src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs

```diff
@@ -323,14 +322,19 @@ private void InitializeHub(THub hub, HubConnectionContext connection)
             hub.Groups = _hubContext.Groups;
         }
 
-        private async Task<bool> IsHubMethodAuthorized(IServiceProvider provider, ClaimsPrincipal principal, IList<IAuthorizeData> policies)
+        private Task<bool> IsHubMethodAuthorized(IServiceProvider provider, ClaimsPrincipal principal, IList<IAuthorizeData> policies)
```

Comment:
nit: This (and the Slow variant) should probably have an `Async` suffix for consistency... though they didn't have that before this PR so 🤷‍♂️